import-dashboards.py

```diff
@@ -24,6 +24,7 @@
 NEW_VERSION_FILE = SCRIPT_DIR + '/VERSION'
 OLD_VERSION_FILE = GRAFANA_DB_DIR + '/PERCONA_DASHBOARDS_VERSION'
 HOST             = 'http://127.0.0.1:3000'
+AUTH_HOST        = 'http://admin:admin@127.0.0.1:3000'
```

Comment:
admin user can be removed on end-user installation, so we can rely only on api_key

Reply:
still actual